var/spack/repos/builtin/packages/py-torch-nvidia-apex/package.py

```diff
@@ -14,11 +14,16 @@ class PyTorchNvidiaApex(PythonPackage, CudaPackage):
     phases = ['install']
 
     version('master', branch='master')
+    # Since the devs don't release their own version numbers,
+    # defining our own. Using the date of the commit referenced
+    # in format YYMMDD, so the number is always increasing.
+    version('201019', commit='8a1ed9e8d35dfad26fb973996319965e4224dcdd')
 
     depends_on('python@3:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
     depends_on('py-torch@0.4:', type=('build', 'run'))
     depends_on('cuda@9:', when='+cuda')
+    depends_on('py-pybind11', when='@201019:,master', type=('build', 'link', 'run'))
```

Comment:
```suggestion
    depends_on('py-pybind11', when='@201019:', type=('build', 'link', 'run'))
```
`master` is > all numbered releases